### ISSUE TITLE
npm run test launching windows virtual machine when on Mac

### ISSUE THREAD

**Issue by mig-25** [garris/BackstopJS#273]
I am on mac os X El Capitan machine with:

```
MacBook-Pro-3:backstopjs sohail$ node -v
v6.4.0
MacBook-Pro-3:backstopjs sohail$ npm -v
3.10.3
```

After following the following steps:

`npm install --save-dev backstopjs`

and 

`npm run genConfig`

I get my backstop_data folder and backstop.json file. Then I run:

`npm run reference`

and I get my screen captures, but running my first test with:

`npm run test`

launches my windows virtual machine to start the browser?????? Thats my first issue.  So after referring to the advice in issue#240 to see if the server is actually running i execute:

`npm run start -- -t 0`

Then:

`npm start`

Then:

`npm run openReport`

But the browser is just not launching localhost:3001/compare/

Whats the cause of this, and why launch the windows virtual machine to the webpage? 

brgds,
Sohail

**Comment by mig-25** [comment]
The strange thing is that it used to at least run the page if I did npm start and npm run openReport, but npm run test would also launch the windows virtual machine on my mac.
Then I checked and found out that Browsersync is already running on http://localhost:3001

**Comment by mig-25** [comment]
I restarted my machine, now at least http://localhost:3001/compare/ is launching.
But both the commands:

`nom run test`
and
`npm run openReport`

launches the windows virtual machine on my mac

**Comment by nmynarcik** [comment]
@mig-25 I had this issue too, where Parallels would launch. To fix this, I pasted the full path to Google Chrome here: https://github.com/garris/BackstopJS/blob/master/gulp/tasks/openReport.js#L18

I don't think this is a backstop issue as much as a OSX/VM issue.

**Comment by mig-25** [comment]
But I have exactly the same code in my openReport.js file. I still starts the windows VM when executing npm run test

**Comment by nmynarcik** [comment]
@mig-25 can you paste your line 18 of `openReport.js`?

**Comment by mig-25** [comment]
Sure:

var options = {
    url: paths.compareReportURL
    ,app: isWin ? "chrome" : "Google Chrome"
  };

**Comment by nmynarcik** [comment]
Can you try:

```
var options = {
    url: paths.compareReportURL
    ,app: isWin ? "chrome" : "/Applications/Google Chrome.app"
  };
```

Change the path of Google Chrome if it is different.

**Comment by mig-25** [comment]
That worked out fine, even though the Google Chrome.app is located in Program folder but could because I am running a Swedish OS maybe. 

Thanks a lot :)

**Comment by nmynarcik** [comment]
👍

**Comment by mig-25** [comment]
Even though the test is running fine and the page launches in OS X I get an error in my terminal after the following:

Testing with  ../../backstop_data/bitmaps_test/compare.json
Opening report ->  http://localhost:3001/compare/

[22:47:07] Finished 'openReport' after 13 ms

npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "test"
npm ERR! node v6.4.0
npm ERR! npm  v3.10.3
npm ERR! code ELIFECYCLE
npm ERR! backstopjs@1.3.5 test: `gulp test`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the backstopjs@1.3.5 test script 'gulp test'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the backstopjs package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     gulp test
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs backstopjs
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls backstopjs
npm ERR! There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/sohail/GitHub/myCoolProject/node_modules/backstopjs/npm-debug.log

**Comment by nmynarcik** [comment]
I get that too. Haven't been able to troubleshoot it yet. There might be another issue for that.

**Comment by garris** [comment]
Should be resolved by #259 .  Please reopen if that is not the case.
